src/NzbDrone.Common.Test/ServiceProviderTests.cs

```diff
@@ -24,8 +25,10 @@ public void Setup()
         [TearDown]
         public void TearDown()
         {
-            WindowsOnly();
```

Comment:
This was causing the test to fail as it was throwing an exception during tear down, not sure if 2.6 didn't care or 3.2 executes the TearDown if the SetUp fails, but this fixes it.